app/containers/container_dropNewPin.js

```diff
@@ -1,7 +1,46 @@
 //this should add ownprops from parent map
-import dropNewPin from '../actions/action_pins.js';
+import React, { Component } from 'react-native';
+import getLocationToSave from '../actions/action_pins.js';
+import Button from 'react-native-button';
 
+// import DropNewPinButton from '../components/DropNewPinButton.js';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+class DropNewPinButton extends Component {
+  constructor(props) {
+    super(props);
+    this.state= {
+     long: null,
+     lat: null,
```

Comment:
Let's use longitude and latitude to be consistent with other data
Also does this duplication of state belong here, or in the store, or in another component?